src/Plugin/GraphQL/Fields/FieldPluginBase.php

```diff
@@ -152,17 +152,18 @@ protected function isLanguageAwareField() {
    * {@inheritdoc}
    */
   protected function resolveDeferred(callable $callback, $value, array $args, ResolveContext $context, ResolveInfo $info) {
-    $result = $this->getRenderer()->executeInRenderContext(new RenderContext(), function () use ($callback, $value, $args, $context, $info) {
-      return $callback($value, $args, $context, $info);
-    });
+    $renderContext = new RenderContext();
+    $result = $callback($value, $args, $context, $info);
```

Comment:
Moving that outside of the render context kinda defeats the whole point, no?

Reply:
In that case, let's move the  line with new RenderContext() down there too.

Reply:
I'm okay with that 😃